cmd/ntw-web/main.go

```diff
@@ -60,6 +59,8 @@ func server(c *cli.Context) error {
 			output = ntw.IntegerToSpanish(number)
 		case "tr", "turkish":
 			output = ntw.IntegerToTurkish(number)
+		case "pt-pt", "portuguesePT":
+			output = ntw.IntegerToTurkish(number)
```

Comment:
should be `ntw.InterToPortuguesePT` I thing

Reply:
yes, my bad

Reply:
IntegerToPortuguesePT